labs/shipped-import-project/1.md

```diff
@@ -0,0 +1,34 @@
+# Shipped - Import Existing project to shipped
+
+1 Choose Exiting Project
+You can select your git repo you want to integrate with shipped, For this example we have sample project 'goproject' written in golang. 
+![](assets/1.PNG)
+
+
+To Integrate your project with shipped you need to have Docker file. this docker file content varies depend on your project requirement.
+
+2 Create a Docker file- This section is specific to the type of project (this project is in Golang)
+
+For Example:
+```Dockerfile
+FROM golang:1.5 // Base Image for Golang Project types.
+EXPOSE 8888     // Port number to expose from docker container to host server.
+WORKDIR /go/src/app  // Workspace directory path where your code exists
+COPY . /go/src/app   // Copy source from host machine to docker container.
+
+// If your projects needs specific build command you can create .shipped directory and place your build/run or test file inside this directory
+
+// Change access permission. that allows docker engine to run these scripts 
+RUN chmod a+x .shipped/build .shipped/run .shipped/test
+
+// Run your build/Run or test script.
+RUN [".shipped/build"]
```

Comment:
In this docker file we should not use .shipped/* scripts. 
Just copy command like 'go build'. Remove  "+RUN chmod a+x .shipped/build .shipped/run .shipped/test" line. 

It would be easy to understand.